src/main/resources/assets/computercraft/lua/rom/help/events.txt

```diff
@@ -11,5 +11,6 @@ Events only on advanced computers:
 "mouse_drag" when a user moves the mouse when held. Arguments are button, xPos, yPos.
 "mouse_up" when a user releases the mouse button. Arguments are button, xPos, yPos.
 "mouse_scroll" when a user uses the scrollwheel on the mouse. Arguments are direction, xPos, yPos.
+"mouse_move" when a user moves the mouse within the terminal. Arguments xPos, yPos. xPos and yPos will be -1, -1 when the mouse is moved outside of the terminal.
```

Comment:
Hahah, this is the first time these docs have been updated in a very long time :D:.